lib/packages/touch.rb

```diff
@@ -0,0 +1,27 @@
+# simple implementation of the 'touch' command, which can be used to create empty text files
+# Initially, this can be used for simple file creation in shellrb. Eventually, I would like to allow for access time manipulation
+class Touch
+  def self.main params
+    # user passes the name of the file to be created as the first parameter
+    new_file = params[0]
+
+    # gets a list of the files and directories currently in the directory
+    ls = `ls`.split("\n")
```

Comment:
I feel it would be better to use the `Ls` package instead. What do you think?

Reply:
That's a great idea, that should make the feature work better cross-platform.